### PR TITLE
Added optional object MGM

### PR DESCRIPTION
**Pull request by freeduerinckx**
To support Member get Member for Mobile Vikings the optional object
mgm has been added.

https://vikingco.atlassian.net/browse/UWA-776

### CODE REVIEW

**Comment by freeduerinckx** [comment]
@daneov , @joen93  @Kjoep just to be sure, this won't break anything on Jim side app/web?
@smaris can you live with these changes?

**Comment by smaris** [comment]
Do we need to specify the errors? https://github.com/vikingco/unleashed-web-api-model/blob/master/paths/subscriptions/subscriptions.yaml#L42

**Comment by daneov** [comment]
✅ - Won't impact us

**Comment by freeduerinckx** [comment]
@smaris good point on the errors. I only expect: referred customer not found? We can make that a 404 (not found) or a 400 (bad request)

**Comment by smaris** [comment]
400 (bad request) is indeed the best choice. 👍

**Comment by freeduerinckx** [comment]
Amended error code

File: definitions/subscriptions/mgm.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+type: object
+title: Member get Member information
+description: |
+  This information allows us to referer to another customer who brought in this new subscription. You can either specify
```
Comment (RoelandMatthijssens) [inline definitions/subscriptions/mgm.yaml]: `referer` -> `refer`

File: definitions/subscriptions/mgm.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  the msisdn or the customer code (exactly one of both).
+properties:
+  msisdn:
+    type: string
+    description: The msisdn of the customer that should reveice an mgm bonus
```
Comment (RoelandMatthijssens) [inline definitions/subscriptions/mgm.yaml]: `reveice` -> `receive`

File: definitions/subscriptions/mgm.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+  customer_id:
+    type: string
+    description: The customer code of the customer that should reveice an mgm bonus
```
Comment (RoelandMatthijssens) [inline definitions/subscriptions/mgm.yaml]: `reveice` -> `receive`

**Comment by freeduerinckx** [comment]
@RoelandMatthijssens I amended the changes

**Pull request merged**